Group-03/Tests/DataTracker.cpp

```diff
@@ -0,0 +1,59 @@
+#include "../../third-party/Catch/single_include/catch2/catch.hpp"
+
+#include "../src/DataTracker.h"
+#include <vector>
+
+
+TEST_CASE("DataTrackerTest 1", "Empty") {
```

Comment:
Style: You should provide a description as the first argument to `TEST_CASE`, and the second argument should be in brackets and be the same for all tests in your file. For example:

```suggestion
TEST_CASE("DataTracker empty", "[DataTracker]") {
```